pkg/azuredisk/azure_controller_common.go

```diff
@@ -72,7 +72,7 @@ const (
 )
 
 var defaultBackOff = kwait.Backoff{
-	Steps:    7,
+	Steps:    20,
```

Comment:
This will cause us to retry from almost 2 hours. Should we be retrying that long?

Reply:
this is actually not related to this fix, there is a timeout set in the outside which would timeout sooner before 20th backoff, and set as 20 steps does not make sense though it won't happen in the beginning.

Reply:
This came in when I reverted Andy's change for waitForDetach commit. @andyzhangx , what should be the correct retry steps here?

Reply:
original value is more reasonable, you could just revert the revert PR.